app/controllers/evidence_hub_controller.rb

```diff
@@ -28,6 +33,7 @@ def form_params
       page: params[:page] || PAGINATION_DEFAULT_PAGE,
       per_page: params[:per_page] || PAGINATION_PER_PAGE
     }
+    form_params[:tag] = params[:tag] if params[:tag].present?
```

Comment:
at what point does `form_params[:tag]` get used? I can only see `params[:tag]` being used through the code in this PR?

Reply:
@willhall88 if I remember correctly, form_params is used for the Search Form and this change was needed to ensure params for the form were persisted.